std/haxe/Int32.hx

```diff
@@ -68,7 +68,7 @@ abstract Int32(Int) from Int to Int {
 
 	@:op(A - B) private static function subFloat(a:Int32, b:Float):Float;
 
-	@:op(A - B) public static function floatSub(a:Float, b:Int32):Float;
+	@:op(A - B) private static function floatSub(a:Float, b:Int32):Float;
```

Comment:
Is this required?
Technically it's a breaking change.

Reply:
? This is a typo (all others 81 operators are private in this Int32 alone, this is the single public one in std for that matter). It is impossible to call this method. What exactly can it break?

Reply:
Ah, indeed. Sorry.